Scripts/ParseEmailFiles/parse_email_files_test.py

```diff
@@ -197,3 +197,13 @@ def executeCommand(name, args=None):
     assert 'test - inner attachment eml.eml' in results[0]['EntryContext']['Email']['Attachments']
     assert isinstance(results[0]['EntryContext']['Email'], dict)
     assert results[0]['EntryContext']['Email']['Depth'] == 0
+
+
+def test_utf_subject_convert():
+    subject = ('[TESTING] =?utf-8?q?=F0=9F=94=92_=E2=9C=94_Votre_colis_est_disponible_chez_votre_co?='
```

Comment:
what about adding some special chars here?

Reply:
This encoding contains special chars. This is how it looks decoded:

![image](https://user-images.githubusercontent.com/1395797/55278165-d4bdb100-5319-11e9-9f96-d2f9335952b4.png)